docs/ProductSummaryShelf.md

```diff
@@ -41,20 +41,18 @@ The Product Summary Shelf is the main block exported by the [Product Summary app
 
-| Prop name        | Type          | Description                | Default value  |
-| :--------------: | :---------: | :--------------------------: | :------------: |
-| `priceBehavior` | `enum` | Whether the component should fetch the most up-to-date price (`async`) or not (`default`). Remember to also set the [Search Result](https://vtex.io/docs/components/content-blocks/vtex.search-result@3.79.1/#configuration)'s`simulationBehavior` prop to `skip` and use the Product Price's [`product-price-suspense`](https://github.com/vtex-apps/product-price/blob/master/docs/README.md) block to render a loading spinner while the price data is being fetched. | `default` |
-| `trackListName` | `boolean` | Whether the component should send the list name to the product page when the product summary is clicked. Disabling it will prevent the `productDetail` GTM event sent on the PDP to identify from which list the user navigated from. | `true` |
+|    Prop name    |   Type    |                                                                                                                                                                                                                                 Description                                                                                                                                                                                                                                 | Default value |
+|:---------------:|:---------:|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:-------------:|
+| `priceBehavior` |  `enum`   | Whether the component should fetch the most up-to-date price (`async`) or (`default`). Remember to configure the [Search Result](https://vtex.io/docs/components/content-blocks/vtex.search-result@3.79.1/#configuration)'s `simulationBehavior` prop to `skip` and use the Product Price [`product-price-suspense`](https://github.com/vtex-apps/product-price/blob/master/docs/README.md) block to render a loading spinner while the price information is being fetched. |   `default`   |
+| `trackListName` | `boolean` |                                                                                                                 Whether the component should send the list name to the product page when the product summary is clicked. Disabling it will prevent the `productDetail` GTM event from being sent on the PDP to identify from which list the user navigated.                                                                                                                 |    `true`     |
```

Comment:
```suggestion
| Prop name | Type | Description | Default value |
| - | - | - | - |
| `priceBehavior` |  `enum` | Determines whether the component should fetch the most up-to-date price (`async`) or (`default`). Remember to configure the [Search Result](https://vtex.io/docs/components/content-blocks/vtex.search-result@3.79.1/#configuration)'s `simulationBehavior` prop to `skip` and use the Product Price [`product-price-suspense`](https://github.com/vtex-apps/product-price/blob/master/docs/README.md) block to render a loading spinner while the price information is being fetched. | `default` |
| `trackListName` | `boolean` | Determines whether the component should send the list name to the product page when the product summary is clicked. Disabling it will prevent the `productDetail` GTM event sent on the PDP to identify from which list the user navigated. | `true` |
```